golem-test-framework/src/config/env.rs

```diff
@@ -461,6 +470,8 @@ impl EnvBasedTestDependencies {
 
         let redis_monitor = redis_monitor_join.await.expect("Failed to join");
 
+        let cassandra = Self::make_cassandra(config.clone());
```

Comment:
I think we want to test all code regardless of if it will be used or not, at the usage level , yes it should be optional, but we want to make sure that it works, so IMO we should test it all the time, or probably I'm missing somthing.

Reply:
This test configuration (`EnvBasedTestDependencies`) is also used for integration and CLI tests, and there we don't want to have cassandra. Having to download it's image and start it would just slow down the already slow CI further.

So please make it lazy initialized, at least, so it only gets created in the storage implementation tests when needed.